### PR TITLE
[FT:]Integrate SMS(third party) alerts

### PR DESCRIPTION
**Pull request by dmaish**
#### Problem
Currently Alerts on a user's job are sent via email.There should be an alternative to this

#### Solution
- Integrate SMS notifications for when a job goes up or down.

#### Screenshots
sms

[https://www.pivotaltracker.com/story/show/158532881]

### CODE REVIEW

File: hc/api/models.py [hunk 1]
```diff
@@ -178,6 +180,8 @@ def send_verify_link(self):
     def transport(self):
         if self.kind == "email":
             return transports.Email(self)
+        elif self.kind == "sms":
```
Comment (simiyu1) [inline hc/api/models.py:183]: Good job!.. check if adding tests for this line will improve your test scores

**Review by simiyu1 — APPROVED** [review]

File: hc/front/views.py [hunk 2]
```diff
@@ -388,6 +388,10 @@ def add_email(request):
     ctx = {"page": "channels"}
     return render(request, "integrations/add_email.html", ctx)
 
+# get sms details
```
Comment (thoth-ky) [inline hc/front/views.py:391]: put this comment inside the function

**Review by thoth-ky — COMMENTED** [review]
FIx coverage

File: hc/api/tests/test_check_model.py [hunk 3]
```diff
@@ -36,3 +36,9 @@ def test_paused_check_is_not_in_grace_period(self):
         self.assertFalse(check.in_grace_period())
 
     ### Test that when a new check is created, it is not in the grace period
```
Comment (miritih) [inline hc/api/tests/test_check_model.py:38]: Use single `#` form comments.

File: requirements.txt [hunk 4]
```diff
@@ -22,3 +22,4 @@ requests==2.9.1
 rjsmin==1.0.12
 six==1.11.0
 whitenoise==3.3.1
+twilio==6.15.1
```
Comment (miritih) [inline requirements.txt:25]: You should have a single line at the end of the file.

**Review by miritih — DISMISSED** [review]

**Review by karanjaE — APPROVED** [review]
Nice work!

**Pull request merged**